imaging.yaml.lock

```diff
@@ -158,12 +159,6 @@ tools:
   - d0960e1b25a8
   - f8f1100d0701
   tool_panel_section_label: Imaging
-- name: 2d_split_binaryimage_by_watershed
```

Comment:
this will not remove the tool, please leave it in. We need to hide this tool from the tool panel.

We need to keep it installed forever, to not break workflows.

Reply:
Like that? d6c007a

Reply:
No please, just don't touch the tools that needs to be deprecated/removed. We can not do this in this repo yet.

Reply:
Got it, 057f900 should be what we need then.

How to remove that tool from the tool panel?